esp_at_lib/src/include/esp/esp_netconn.h

```diff
@@ -75,6 +75,9 @@ int8_t          esp_netconn_getconnnum(esp_netconn_p nc);
 void            esp_netconn_set_receive_timeout(esp_netconn_p nc, uint32_t timeout);
 uint32_t        esp_netconn_get_receive_timeout(esp_netconn_p nc);
 
+espr_t          esp_netconn_connect_ex(esp_netconn_p nc, const char* host, esp_port_t port,
+                                       uint16_t keep_alive, const char* local_ip, esp_port_t local_port, uint8_t mode);
+
```

Comment:
Good naming, I have to correct mistake of `esp_conn_startex` and rename it to `esp_conn_start_ex` instead.